client/components/TestQueueRow/index.jsx

```diff
@@ -530,6 +530,11 @@ const TestQueueRow = ({
         <LoadingStatus message={loadingMessage}>
             <tr className="test-queue-run-row">
                 <th>{renderAssignedUserToTestPlan()}</th>
+                <td style={{ padding: '0.75rem' }}>
```

Comment:
We don't do these inline styles but considering this is only temporary patch, it's fine!